prog/vm/nexus.rb

```diff
@@ -240,7 +238,7 @@ def before_run
     case host.sshable.cmd("common/bin/daemonizer --check prep_#{q_vm}")
     when "Succeeded"
       host.sshable.cmd("common/bin/daemonizer --clean prep_#{q_vm}")
-      vm.nics.each { _1.incr_setup_nic }
```

Comment:
Likewise these hunks are also not related to the stated patch purpose right?

Reply:
I moved them